containers/Containerfile.acmesolver

```diff
@@ -0,0 +1,16 @@
+ARG BASE_IMAGE
```

Comment:
Seems odd to have a dedicated root directory for this. Would this not be better to live under `/hack/containers`?

Reply:
I think a dedicated root dir is reasonably justified here; our containers are our most important release artifacts and I think having our Containerfiles be fairly prominent is reasonable; plus there are tonnes of projects who have their Dockerfile in the root of the their repo.

I'd probably vote to move our helm chart template into `/helm` or to create `/release/{helm,containers}` in the longer term.

It doesn't matter though, I'll just move them into hack